docs/ReleaseNotes.md

```diff
@@ -24,7 +24,7 @@ In this realase, the various implementations of the `RecordQueryPlan` interface
 * **Performance** Improvement 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 5 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Feature** Feature 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Feature** Permuted grouped aggregate [(Issue #1085)](https://github.com/FoundationDB/fdb-record-layer/issues/1085)
```

Comment:
I am open to suggestions for a better name for this type of index.

Reply:
Hmm. Well, a `VALUE` index can do `MIN` and `MAX`, as they are at the ends of a range. I guess having no permutation could mean skip the secondary keyspace maintenance, but then the index isn't good for other kinds of queries.

I'm inclined to have the index reject things that aren't useful rather than just things that aren't possible. But maybe I'm not thinking about it right.